src/main/java/cat/udl/eps/softarch/learnphysics/repository/LevelRepository.java

```diff
@@ -1,9 +1,17 @@
 package cat.udl.eps.softarch.learnphysics.repository;
 
 import cat.udl.eps.softarch.learnphysics.domain.Level;
+import cat.udl.eps.softarch.learnphysics.domain.Topic;
 import org.springframework.data.repository.PagingAndSortingRepository;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
 
-@RepositoryRestResource
 public interface LevelRepository extends PagingAndSortingRepository<Level, Integer> {
+    Optional<Level> findById(@Param("levelId") String id);
```

Comment:
Since the `id` of Level is an Integer, this function should receive an Integer instead of a String.